analyses/TN_cd_2000/doc_TN_cd_2000.md

```diff
@@ -0,0 +1,23 @@
+# 2000 Tennessee Congressional Districts
+
+## Redistricting requirements
+In Tennessee, districts must:
```

Comment:
Sorry, I made this update after your PR, but can we link to the rules archive if that's what you used? 

```
according to [NCSL Redistricting Law 2000](https://web.archive.org/web/20041216185957/https://www.senate.mn/departments/scr/redist/red2000/Tab5appx.htm),
```